README.md

```diff
@@ -1,29 +1,26 @@
-# Open Permissions Platform Offer Viewer
-This library is responsible for displaying an ODRL offer in human-readable form.
+# Offer Viewer
+A javascript component that will display an ODRL offer in human-readable form.
```

Comment:
Would be nice if you started the doc with a table of acronyms, even if it just has the one entry ODRL.
See https://github.com/openpermissions/support-docs/blob/master/documents/markdown/versioning/api-versioning.md
as an example we use in support docs